### PR TITLE
REFACTOR: merge `git grab` and `clone`

### PR DESCRIPTION
**Pull request by amtoine**
this PR
- merges `git grab` and `clone` together
- removes the `log` commands which are not used anymore (and we can use the standard library :smirk:)
- `root_dir` now only computes the root dir without subdirectory, i do that with `path join` only

> **Note**
> some features to take care of
> - more logging?
> - what happens when the repo already exists? `cd` into it? update it with `-u`?
> - do not put the repos under `$host` with an option / config

### CODE REVIEW

File: mod.nu [hunk 1]
```diff
@@ -37,78 +27,24 @@ export def-env "git grab select" [] {
 # TODO: add support for other hosts than github
 # TODO: better worktree support
 
-# Clone a github repository into a standard location organised by domain and path.
-export def-env "git grab" [
-    owner,          # Either the owner, your repo name, or "owner/repo"
-    repo?: string,  # the repo name
-    --ssh           # use ssh instead of https
-    --bare(-b)      # use a bare repo (specific to worktrees)
+export def "git grab" [
```
Comment (melMass) [inline mod.nu:30]: I think we should at least have a minimal comment/doc line above all exported defs

**Comment by melMass** [comment]
Awesome, much cleaner, I would just keep minimal comments at least

File: mod.nu (hunk 1 above)
Comment (amtoine) [reply mod.nu:30]: oooh yes, sorry, forgot about that!

i add it right now :+1:

Comment (amtoine) [reply mod.nu:33]: is it better with 0b85b0a? :yum:

**Review by melMass — APPROVED** [review]
🚀

**Pull request merged**